tests/bluetooth/tester/src/btp_vcp.c

```diff
@@ -0,0 +1,580 @@
+#include <zephyr/bluetooth/audio/vcp.h>
+#include <zephyr/bluetooth/audio/aics.h>
+#include "zephyr/bluetooth/audio/vocs.h"
+#include "zephyr/sys/util.h"
+
+#include <app_keys.h>
+
+#include <zephyr/logging/log.h>
+#define LOG_MODULE_NAME bttester_vcp
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#include "btp/btp.h"
+
+#define CONTROLLER_INDEX 0
+#define BT_AICS_MAX_INPUT_DESCRIPTION_SIZE 16
+#define BT_AICS_MAX_OUTPUT_DESCRIPTION_SIZE 16
+
+struct bt_vcp_vol_rend_register_param vcp_register_param;
+struct bt_vcp_included included;
+
+static void set_register_params(uint8_t gain_mode);
+uint8_t tester_init_vcp(void);
+
+
+/* Volume Control Service */
+static void vcs_supported_commands(uint8_t *data, uint16_t len)
+{
+	struct net_buf_simple *buf = NET_BUF_SIMPLE(BTP_DATA_MAX_SIZE);
+
+	net_buf_simple_init(buf, 0);
+	net_buf_simple_add_u8(buf, VCS_READ_SUPPORTED_COMMANDS);
+	net_buf_simple_add_u8(buf, VCS_SET_VOL);
+	net_buf_simple_add_u8(buf, VCS_VOL_UP);
+	net_buf_simple_add_u8(buf, VCS_VOL_DOWN);
+	net_buf_simple_add_u8(buf, VCS_MUTE);
+	net_buf_simple_add_u8(buf, VCS_UNMUTE);
+
+	tester_send(BTP_SERVICE_ID_VCS, VCS_READ_SUPPORTED_COMMANDS,
+		    CONTROLLER_INDEX, buf->data, buf->len);
+}
+
+static void set_volume(uint8_t *data)
+{
+	const struct vcs_set_vol_cmd *cmd = (void *)data;
+	uint8_t volume;
+
+	volume = cmd->volume;
+
+	LOG_DBG("Set volume 0x%02x", volume);
+
+	if (bt_vcp_vol_rend_set_vol(volume) != 0) {
+		tester_rsp(BTP_SERVICE_ID_VCS, VCS_SET_VOL,
+			   CONTROLLER_INDEX, BTP_STATUS_FAILED);
+		return;
+	}
+
+	tester_rsp(BTP_SERVICE_ID_VCS, VCS_SET_VOL, CONTROLLER_INDEX,
+		   BTP_STATUS_SUCCESS);
+}
+
+static void vol_up(void)
+{
+	LOG_DBG("Volume Up");
+
+	if (bt_vcp_vol_rend_vol_up() != 0) {
+		tester_rsp(BTP_SERVICE_ID_VCS, VCS_VOL_UP,
+			   CONTROLLER_INDEX, BTP_STATUS_FAILED);
+		return;
+	}
+
+	tester_rsp(BTP_SERVICE_ID_VCS, VCS_VOL_UP, CONTROLLER_INDEX,
+		   BTP_STATUS_SUCCESS);
+}
+
+static void vol_down(void)
+{
+	LOG_DBG("Volume Down");
+
+	if (bt_vcp_vol_rend_vol_down() != 0) {
+		tester_rsp(BTP_SERVICE_ID_VCS, VCS_VOL_DOWN,
+			   CONTROLLER_INDEX, BTP_STATUS_FAILED);
+		return;
+	}
+
+	tester_rsp(BTP_SERVICE_ID_VCS, VCS_VOL_DOWN, CONTROLLER_INDEX,
+		   BTP_STATUS_SUCCESS);
+}
+
+static void mute(void)
+{
+	LOG_DBG("Mute");
+
+	if (bt_vcp_vol_rend_mute() != 0) {
+		tester_rsp(BTP_SERVICE_ID_VCS, VCS_MUTE, CONTROLLER_INDEX,
+			   BTP_STATUS_FAILED);
+		return;
+	}
+
+	tester_rsp(BTP_SERVICE_ID_VCS, VCS_MUTE, CONTROLLER_INDEX,
+		   BTP_STATUS_SUCCESS);
+}
+
+static void unmute(void)
+{
+	LOG_DBG("Unmute");
+
+	if (bt_vcp_vol_rend_unmute() != 0) {
+		tester_rsp(BTP_SERVICE_ID_VCS, VCS_UNMUTE, CONTROLLER_INDEX,
+			   BTP_STATUS_FAILED);
+		return;
+	}
+
+	tester_rsp(BTP_SERVICE_ID_VCS, VCS_UNMUTE, CONTROLLER_INDEX,
+		   BTP_STATUS_SUCCESS);
+}
+
+static void vcs_state_cb(int err, uint8_t volume, uint8_t mute)
+{
+	if (err) {
+		LOG_DBG("VCP state cb err (%d)", err);
+	}
+	LOG_DBG("VCP state cb");
```

Comment:
Is there any reason to not just have 
```suggestion
	LOG_DBG("VCP state cb err (%d)", err);
```
?

Reply:
Done